core/src/main/resources/org/elasticsearch/tasks/task-results-index-template.json

```diff
@@ -0,0 +1,52 @@
+{
+  "template" : ".results",
+  "order" : 2147483647,
+  "settings" : {
+    "number_of_shards" : 1,
+    "number_of_replicas" : 1,
```

Comment:
Both of these seem like something that'd be useful to tune in really esoteric use cases. I argue 2 or 3 replicas is a better default though because we expect the index to be small in the usual use case and we expect people to really want to have their results when they need them.